ClangAutoComplete.py

```diff
@@ -11,53 +11,53 @@ class ClangAutoComplete(sublime_plugin.EventListener):
 	compl_regex = re.compile("COMPLETION: ([^ ]+) : ([^\\n]+)")
 	file_ext = re.compile("[^\.]+\.([^\\n]+)")
 	syntax_regex = re.compile("\/([^\/]+)\.(?:tmLanguage|sublime-syntax)")
-	project_name_regex = re.compile("([^\.]+).sublime-project")
 	settings_time = 0
 
+	def to_bool(self, str_flag):
+		return str_flag == "true"
+
 	def load_settings(self):
 		# Only load the settings if they have changed
-		settings_modified_time = path.getmtime(sublime.packages_path()+"/ClangAutoComplete/"+"ClangAutoComplete.sublime-settings")
+		settings_modified_time = path.getmtime(
```

Comment:
It seems to reload the plugin every time I change the user settings, but you're right, I wouldn't bet my life on it.